requirements/test.txt

```diff
@@ -8,7 +8,6 @@ flake8>=3.9.2
 pre-commit>=1.0
 pytest-forked
 sklearn
```

Comment:
This entry seems a duplication of a below line `scikit-learn>0.22.1`. Can be removed in this PR?
```suggestion
```